packages/application-shell/package.json

```diff
@@ -102,13 +101,14 @@
   },
   "devDependencies": {
     "@commercetools-frontend/ui-kit": "4.0.0",
-    "react": "16.6.3",
+    "prop-types": "15.6.2",
```

Comment:
why is this necessary?

Reply:
I found it super odd to have it as a dependency. Prop-types to me are always a devDep?

Reply:
My understanding is that as `prop-types` are imported into the production code, they should be regular dependencies (just like lodash etc which is also imported).

Unless we use a compile step to remove the imports - but even then nothing would import the prop-types and that would be fine to have them as a regular dependency as well.

Reply:
Ok. Reverted in last commit.